src/include/sof/math/numbers.h

```diff
@@ -71,7 +71,7 @@ int32_t find_max_abs_int32(int32_t vec[], int vec_length);
  */
 int norm_int32(int32_t val);
 
-uint32_t crc32(const void *data, uint32_t bytes);
+uint32_t crc32(uint32_t crc32, const void *data, uint32_t bytes);
```

Comment:
I'd call the passed initial crc something else, like "previous", "base" or "start"